malaria24/ona/tests/test_models.py

```diff
@@ -42,7 +52,7 @@ def mk_case(self, **kwargs):
             'first_name': 'first_name',
             'last_name': 'last_name',
             'locality': 'locality',
-            'date_of_birth': 'date_of_birth',
+            'date_of_birth': self.mk_random_date(),
```

Comment:
What we're getting from Ona is a string, not a timestamp that's exactly like this. Perhaps that's better to test with?